frontend/src/citizen-frontend/calendar/DayElem.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styled from 'styled-components'
+import colors from 'lib-customizations/common'
+import { formatDate } from 'lib-common/date'
+import LocalDate from 'lib-common/local-date'
+import {
+  FixedSpaceColumn,
+  FixedSpaceRow
+} from 'lib-components/layout/flex-helpers'
+import { useTranslation } from '../localization'
+import { DailyReservationData } from './api'
+
+export interface DayProps {
+  data: DailyReservationData
+}
+
+export default React.memo(function DayElem({
+  data: { date, isHoliday, reservations }
+}: DayProps) {
+  const i18n = useTranslation()
+
+  return (
+    <DayDiv alignItems="center" today={date.isToday()} holiday={isHoliday}>
+      <DayColumn spacing="xxs">
+        <div>
+          {i18n.common.datetime.weekdaysShort[date.getIsoDayOfWeek() - 1]}
+        </div>
+        <div>{date.format('dd.MM.')}</div>
+      </DayColumn>
+      <div>
+        {reservations
+          .map(
+            (reservation) =>
+              `${formatDate(reservation.startTime, 'HH:mm')} - ${formatDate(
+                reservation.endTime,
+                'HH:mm'
+              )}`
+          )
+          .join(', ')}
+      </div>
+      {date.isBefore(LocalDate.today()) && <HistoryOverlay />}
+    </DayDiv>
+  )
+})
+
+const DayColumn = styled(FixedSpaceColumn)`
+  width: 48px;
```

Comment:
Would a margin work better instead of a fixed width?

Reply:
I think width is better here to make sure that the next "column" is aligned

Reply:
Yeah, I didn't think about the fact that different dates have different widths because the font isn't monoscape 😄

In that case I would use `rem` units to define the width.